### ISSUE TITLE
Call for feedback

### ISSUE THREAD

**Issue by jerrinot** [hazelcast-guides/guides-site#5]
As a guide reader I would like to know how to report typos/feedback/any_other_issue
Given the guides are maintained on github it's trivial to send a PR with a fix. What's not trivial is to find the right repo. 

I can imagine a ribbon in a corner, or something simple like the "Send Us a Fix" prompt in a [reference manual](https://docs.hazelcast.org/docs/4.0.2/manual/html-single/index.html)

![image](https://user-images.githubusercontent.com/158619/90144588-47002b80-dd7f-11ea-8587-7676fb90febe.png)

**Comment by enozcan** [comment]
There exists an `edit` section on the guides page, which navigates to edit page of the current content. Do you mean that or something more specific?

<img width="1159" alt="Screen Shot 2020-08-13 at 17 20 51" src="https://user-images.githubusercontent.com/44522401/90146257-7e73d580-dd89-11ea-93f5-1c5f4286ff43.png">

**Comment by jerrinot** [comment]
exactly. except something what an old person - like myself - can actually see! :)

**Comment by enozcan** [comment]
You are right, it's not really visible on the page. An edit button would be better and I will work on it. Thanks for reporting.
